sections/executive-team-section/executive-avatar.js

```diff
@@ -34,6 +34,7 @@ const Container = styled.div`
 const AvatarContainer = styled(MediaContainer)`
   width: 100%;
   max-width: 160px;
+  padding: 0 5px;
```

Comment:
This causes the pictures to be smaller than they are in the designs. The solution here is to make the container larger, not to add padding to the images. Also, please ensure to apply the changes only to mobile.

Reply:
At mobile resolutions, the max-width of the AvatarContainer is 160px while the Container width is set to 50% so, at some point (ie, these resolutions), the avatar will completely fill the Container unless we reduce the padding on AvatarList (the Container's container). However, reducing the AvatarList padding doesn't prevent the problem from occurring - it just happens at a lower resolution. I think we are stuck with adding padding to the AvatarContainer (or the Container).